lib/msal-node/src/cache/serializer/SerializerTypes.ts

```diff
@@ -3,14 +3,18 @@
  * Licensed under the MIT License.
  */
 
-import { StringDict, AccountCache, IdTokenCache, AccessTokenCache, RefreshTokenCache, AppMetadataCache } from "@azure/msal-common";
+import {AccountCache, IdTokenCache, AccessTokenCache, RefreshTokenCache, AppMetadataCache } from "@azure/msal-common";
+
+export type Dict = {
+    [key: string]: object;
```

Comment:
Also, can this be `Record<string, object>`?

Reply:
Is that an equivalent statement? Where is `Record` defined?

Reply:
Yes, those should be equivalent. It's a built in TS type.

Reply:
`Record<string, object>` is now used as appropriate and `Dict` is removed